### DISCUSSION TITLE
GitHub followers and stars

### DISCUSSION THREAD

**Discussion by SantinoMajur** [community/community#20181] [Other Feature Feedback, Questions, & Ideas]
Hey folks, how do I get followers in GitHub and stars on my repository.

**Comment by D7EAD** [comment]
You can gain traction on your repositories and account by sharing both on social media sites, such as Reddit, Twitter, etc. You could also gain traction with the help of blogs (such as Medium). Frankly, though, if one were to create repositories impressive enough, sometimes traction comes naturally!

**Reply by QuocHuy03** [reply]
Response #1

**Reply by NeenGame** [reply]
Response #2

**Comment by evan-hwang** [comment]
It's a dilemma for me too. Why don't we be each other's followers?

**Reply by SantinoMajur** [reply]
that would be greate @evan-hwang

**Reply by kelvzzy** [reply]
@kelvzzy

**Reply by U7P4L-IN** [reply]
i don't say anything.bcz i have lot of followers

**Comment by radar-blip** [comment]
Its up to other members to follow you and your work. Same goes for starring your repositories. 
If your work is exciting and useful to others, the more attention it will get.
Hang in there and keep coding!

**Reply by SantinoMajur** [reply]
Easy, thanks mate

**Reply by radar-blip** [reply]
Anytime.

**Reply by Sajib-Bhattacharjee** [reply]
Great.

**Comment by prathikshetty2002** [comment]
> Hey folks, how do I get followers in GitHub and stars on my repository.

There we go again. If you want to increase your github engagement and reach you need to share your work with others. You can do this by sharing it in communities , network with individuals and promote it in open source groups. Birds of feather flock together :)

**Reply by Sajib-Bhattacharjee** [reply]
Please visit my profile to get useful documentation

**Comment by myConsciousness** [comment]
@SantinoMajur 

First of all, to get followers and stars on GitHub, you need novel and creative products. Unlike other recreational social networking sites, engineers are very demanding. 

However, the excellence of the products correlates with the number of stars, but not the number of followers. There are many people who maintain great repositories but have a low follower count.

And the best way for an unknown developer to get people to know the product he created is to promote it in different ways: on huge platforms like Twitter, Reddit, etc :)

**Reply by U7P4L-IN** [reply]
I need 1k Github follower.anyone help me?

**Reply by Sajib-Bhattacharjee** [reply]
Please visit my profile to get useful documentation

**Comment by jfullstackdev** [comment]
it's about increasing visibility, for me Twitter and GitHub, with those links linked at one another, I will initiate following both on GitHub and Twitter,

back then I was just waiting but why not take the initiative, so I now have followers 3.1k on GitHub and 2.5k on Twitter

you must be sure that you have a landing page that can be trusted and active

**Reply by michael-azogu** [reply]
how did you grow on Twitter. was it credentials or content, some people are just famous while some have to create value on the platform through threads & spaces

**Reply by jfullstackdev** [reply]
I started following users that are worth it and they followed me back, I'm not famous, maybe just the content and my achievements being posted, I even pinned one of my achievements

**Reply by Sajib-Bhattacharjee** [reply]
Please visit my profile to get useful documentation

**Comment by codewithmuh** [comment]
Push good and creative Content, and Help others! Then you will get this.

**Reply by Sajib-Bhattacharjee** [reply]
Please visit my profile to get useful documentation

**Comment by melvincwng** [comment]
@SantinoMajur, try contributing to more open source projects. This can help improve your visibility on GitHub.

Hope this helps :)

**Reply by Sajib-Bhattacharjee** [reply]
Please visit my profile to get useful documentation

**Comment by Nirzak** [comment]
Try contributing more to open-source projects. Be precise and try to make a good readme profile. Upload or create more useful projects. Hopefully, you can gain a good number of followers.

**Reply by Sajib-Bhattacharjee** [reply]
Please visit my profile to get useful documentation

**Comment by globart** [comment]
Hey, let's follow each other

**Reply by U7P4L-IN** [reply]
Bro,Can you help me? How can get sponsors achievement

**Reply by globart** [reply]
you have to sponsor someone atleast once

**Reply by U7P4L-IN** [reply]
bro it is free or paid which?

**Reply by globart** [reply]
you can make 1 time donation of 1 dollar to someone, that's the minimum

**Reply by Sajib-Bhattacharjee** [reply]
Please visit my profile to get useful documentation

**Comment by Tanmay-Somani** [comment]
> Hey folks, how do I get followers in GitHub and stars on my repository.

Gaining followers on GitHub and stars on your repository involves building a strong online presence, creating valuable projects, and actively engaging with the GitHub community just like what you did now by asking a question. Try to improve your readme, and also do a lot of open source honestly, and start promoting your github in person to fellow github developers of yours.

**Reply by U7P4L-IN** [reply]
Agree❤️

**Reply by Nirzak** [reply]
Agreed and upvoted!

**Reply by Sajib-Bhattacharjee** [reply]
Please visit my profile to get useful documentation

**Comment by U7P4L-IN** [comment]
I went to get github sponsorship button.Anyone have Expart? JUST SAY

**Reply by Tanmay-Somani** [reply]
![image](https://github.com/community/community/assets/98737069/ac4d64ce-5ea4-4f1e-bfc8-8f64e19d8b6a) 
Go to Settings in your specific Repository or your main README.md

**Reply by U7P4L-IN** [reply]
No i try to show sponsor button on profile

**Reply by Tanmay-Somani** [reply]
I'm sorry, but that isn't possible. However, you can perform the same task for your special repository

**Reply by Sajib-Bhattacharjee** [reply]
Please visit my profile to get useful documentation

**Comment by nolanfoster123** [comment]
Getting followers on GitHub and stars on your repositories primarily involves building a strong presence, contributing valuable content, and engaging with the GitHub community. Here are some effective strategies to help you achieve this:

- Contribute Actively: One of the best ways to gain visibility and followers is by actively contributing to open-source projects or repositories related to your interests. This showcases your skills and interests to the community.
- Create High-Quality Repositories: When you create a new repository, ensure that it's well-documented, has a clear README, and follows best practices for code organization. A well-maintained project is more likely to attract stars.
- Engage in Discussions: Participate in discussions, raise issues, and contribute to existing discussions on GitHub. This helps you connect with other developers and gain recognition.
- 
- Follow Others: Follow developers whose work you admire or those who work in your field of interest. Many of them will follow you back.
- Promote Your Work: Share your GitHub projects on social media, developer forums, and other relevant platforms. Describe why your project is valuable and how it can benefit others.
- Collaborate: Collaborate with others on interesting projects. This not only helps you learn but also exposes your work to a wider audience.
- Use Meaningful Tags: Use relevant tags and keywords in your repositories and descriptions to make them more discoverable through GitHub's search.
- Respond to Feedback: When someone opens an issue or submits a pull request on your repository, be responsive and respectful. A positive interaction can lead to stars and followers.
- Showcase Your Expertise: Regularly push code to your repositories to show that you're actively working on projects. A consistent track record can attract more attention.
- Participate in Hackathons and Challenges: Join GitHub-hosted hackathons and coding challenges. These events can boost your visibility and provide opportunities for collaboration.
- Write Informative READMEs: A well-written README not only explains your project but also makes it more appealing to potential contributors and users.
- Network at GitHub Events: Attend GitHub events, webinars, and conferences. Networking with other developers can lead to more followers.
- Be Patient and Consistent: Building a strong GitHub presence takes time. Be patient, stay consistent, and continue to improve your skills and projects.

Remember that gaining [Twitter followers](https://curiousblogger.com/twitter-unfollow-tools/) and stars on GitHub is a gradual process. It's important to focus on the quality of your contributions and engagement rather than just the quantity. Building a positive reputation in the developer community will naturally attract more attention to your work.

**Comment by laurentiucozma12** [comment]
Sharing your profile might work. But it would be better if you start an open-source project and ask people to contribute. Also, share the project on social media so many people would see it and come to see your project. And that's how you can get a lot of stars if the project is really good or interesting.

**Reply by Sajib-Bhattacharjee** [reply]
Please visit my profile to get useful documentation

**Comment by U7P4L-IN** [comment]
𝙄 𝙖𝙡𝙧𝙚𝙙𝙮 𝙝𝙖𝙫𝙚𝙚 1.1𝙠+ 𝙁𝙤𝙡𝙡𝙤𝙬𝙚𝙧😊

**Reply by psalmsin1759** [reply]
please how do i get followers? trying to grow my followers

**Reply by U7P4L-IN** [reply]
Follwer Depends on Your Repository style

**Reply by Sajib-Bhattacharjee** [reply]
Please visit my profile to get useful documentation

**Comment by ehsantg** [comment]


**Reply by U7P4L-IN** [reply]
Really?

**Reply by Sajib-Bhattacharjee** [reply]
Please visit my profile to get useful documentation

**Reply by MuhamedHabib** [reply]
how do I get followers in GitHub and stars on my repository.

**Reply by U7P4L-IN** [reply]
followers depens on your project and repo files. so try good quailty poject making

**Comment by Sajib-Bhattacharjee** [comment]
Please visit my profile to get useful documentation.

**Comment by NicholasGannon** [comment]
Hi everyone,

There's some great information within this thread!

If anyone's interested, I'm currently working on a repository that provides a curated list of useful **font and typography** resources for designers/developers! I'd really appreciate if you could check it out and give it a star!

https://github.com/NicholasGannon/Font-Resources

Thank you

**Comment by Amar-HR** [comment]
Dear All, 

Can you please leave a star on our product. Thanks in advance. 

https://github.com/raga-ai-hub/agentneo

**Comment by kesjien** [comment]
Forecasting tool https://github.com/kesjien/wavexplain
